docs/components/checkboxes.md

```diff
@@ -21,11 +21,16 @@ Use checkboxes to:
 
 ## Contents
 
-- [Contents](#contents)
-- [Using checkboxes](#using-checkboxes)
-- [Checkboxes](#checkboxes)
-- [Checkbox key properties](#checkbox-key-properties)
-- [Theming](#theming)
+- [Selection controls: checkboxes](#selection-controls-checkboxes)
```

Comment:
This change looks incorrect. I think we are only supposed to include top level headers, and `contents` is still in here.

Reply:
Done